userguide/services.rst

```diff
@@ -238,21 +238,22 @@ Licensed Asigra Backup software can use %brand% as the storage backend.
 For the initial backend configuration, click
 :menuselection:`Services --> Asigra DS-System`. When prompted to choose
 the :guilabel:`Base Filesystem`, select the dataset to store the Asigra
-backups, then click :guilabel:`OK`. Any required database entries are created and the service is started.
+backups, then click :guilabel:`OK`. Any required database entries are
+created and the service is started.
 
 .. note:: Asigra DS-Operator requires a working installation of
    `Java JRE <https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html>`__
    and a security exception for the %brand% system. To add the exception,
    use :menuselection:`Configure Java --> Security --> Edit Site List --> Add`
    and enter the URL to the %brand% system. If the browser prompts for
```

Comment:
"If the browser prompts for the application to open :file:`DSOP.jnlp` with,..." is confusing/doesn't make sense. What about:
```suggestion
   and enter the URL to the %brand% system. If the browser prompts
```